### PR TITLE
PT-1221 | Fix swedish translations

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

Fix swedish translations

### Closes :no_good_woman:

**[PT-1221](https://helsinkisolutionoffice.atlassian.net/browse/PT-1221):**

### CODE REVIEW

File: public/locales/sv/enrolment.json [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-    "labelIsSameResponsiblePerson": "Samma som anmälare",
-    "labelIsSharingDataAccepted": "Jag accepterar att dela min information med arrangören",
+    "labelIsSameResponsiblePerson": "Samma som anmälaren",
+    "labelIsSharingDataAccepted": "Jag godkänner att mina uppgifter delas med arrangören",
     "labelLanguage": "Språk",
-    "labelNotificationLanguage": "Språket som används i påminnelser",
+    "labelNotificationLanguage": "Språket för meddelanden",
     "person": {
       "labelEmailAddress": "E-postadress",
       "labelName": "Name",
```
Comment (Jokinen) [inline public/locales/sv/enrolment.json:18]: Should this be updated as well?

File: public/locales/sv/enrolment.json [hunk 2]
```diff
@@ -2,17 +2,17 @@
   "buttonBack": "Tillbaka",
   "enrolmentForm": {
     "buttonSubmit": "Skicka in registrering",
-    "buttonSubmitEnquiry": "Skicka en Bokningsförfrågan",
+    "buttonSubmitEnquiry": "Skicka anmälan",
     "infoText1": "Vi använder inte den information du tillhandahåller för marknadsföringsändamål.",
```
Comment (Jokinen) [inline public/locales/sv/enrolment.json:6]: Should this be updates as well?

**Review by Jokinen — APPROVED** [review]
I checked the document in the ticket and found a few places where there may be a mismatch.

This line in the translation doc is a bit unclear as it lists two keys and may be a reason for the confusion:

>**enrolmentForm.infoText1** _privacyPolicy_, Vi använder inte de uppgifter som du uppgett för marknadsföring.

Looks good toherwise.

File: public/locales/sv/common.json [hunk 3]
```diff
@@ -45,8 +45,8 @@
     },
     "buttonClose": "Stäng"
   },
-  "privacyPolicy": "Integritetspolicy",
-  "providerUI": "Åt evenemangsarrangören",
+  "privacyPolicy": "Vi använder inte de uppgifter som du uppgett för marknadsföring.",
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply public/locales/sv/common.json:48]: Good catch!

File: public/locales/sv/enrolment.json (hunk 1 above)
Comment (Sartonon) [reply public/locales/sv/enrolment.json:18]: yes :)

**Pull request merged**